dias/tracker.py

```diff
@@ -309,6 +314,10 @@ def new_files(self, dias_task_name, filetypes, update=True):
             list of filetypes of interest
         update : Boolean
             If true, updates the File Table with the current state of `self.path`.
+        start : datetime.datetime or unix float
```

Comment:
Isn't it `float, datetime or None`, where `float` has to be a unix timestamp?

Reply:
That is better, yes! I will change it